packages/@okta/vuepress-site/docs/guides/authenticators-google-authenticator/main/java/challengeintegrationsummary.md

```diff
@@ -0,0 +1,7 @@
+<div class="common-image-format">
+
+![Sequence diagram showing Google challenge summary](/img/authenticators/dotnet-authenticators-google-challenge-summary.png)
+
+TODO: Need a Java version of this
```

Comment:
Once the diagram is updated for java, add your diagram here: https://www.figma.com/file/YH5Zhzp66kGCglrXQUag2E/%F0%9F%8C%9F-Updated-Diagrams-for-Dev-Docs?node-id=709%3A140